src/components/AnimatedButton/style.ts

```diff
@@ -9,16 +9,15 @@ const useStyles = makeStyles(() => ({
     textTransform: 'none',
     ...newTypography.body1,
     color: colors.invariant.componentBcg,
-    background: colors.invariant.pinkLinearGradientOpacity,
+    background: colors.invariant.greenLinearGradient,
```

Comment:
are you sure this won't affect any other places, where this animated button is used?